resources/manifests/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
+Main-Class: evaluation.GameReport
```

Comment:
Let's keep this out of repo to avoid build conflicts.

Reply:
Completely! Careless of me to let that sneak in.